extensions/ql-vscode/src/pure/interface-types.ts

```diff
@@ -53,9 +61,11 @@ export interface Interpretation {
    * they appear in the sarif file.
    */
   sortState?: InterpretedResultsSortState;
-  sarif: sarif.Log;
+  data: T;
 }
 
+export type Interpretation = InterpretationT<InterpretationData>;
```

Comment:
```suggestion
export type Interpretation = Interpretation<InterpretationData>;
```

Reply:
This doesn't appear to work, I guess you can't have both a generic type and a non-generic type with the same name?